### ISSUE TITLE
Compass doesn't get enchanted if tracked player is already in the nether

### ISSUE THREAD

**Issue by SeerLite** [yummypasta/MinecraftManhunt#39]
## Describe the bug
The compass doesn't get its "enchanted color effect" if a hunter starts tracking a speedrunner when they're already in the nether. This can be really confusing (was to me and my friends) because it gives the impression that it's not working, even though it is.

## Potential Causes
I have Multiverse-Core and Multiverse-NetherPortals installed too, and I haven't checked if it happens without them installed (I'm pretty sure it does though).

## Environment Info
**Plugin version:** 1.3.3.2 

**Paper server version:** git-Paper-329

## Configuration file
Sending us your plugin `config.yml` file may help in diagnosing the issue. Paste the contents of the plugin's config file here: (feel free to omit your Discord token for security reasons)
```yaml
# How long the hunters should get blindness and slowness when the match starts.
headStartDuration: 5
compassEnabledInNether: true
sendUsageData: true
enableDiscord: false
discordToken: XXXXX
ip: a minecraft server
processDiscordCommands: true
discordServerId: XXXXX
musicChannelId: XXXXX
hunterRoleId: XXXXX
runnerRoleId: XXXXX
spectatorRoleId: XXXXX
```
(I left UUID out)

## Server Logs
Please check your Server Console for any warnings or errors created by the plugin. If any are any, please copy and paste them here:
```
[17:42:50] [Server thread/WARN]: [MinecraftManhunt] Task #10189 for MinecraftManhunt v1.3.3.2 generated an exception
java.lang.IllegalArgumentException: Name cannot be null
at org.apache.commons.lang.Validate.notNull(Validate.java:192) ~[minecraft_server.jar:git-Paper-329]
at org.bukkit.craftbukkit.v1_16_R3.CraftServer.getPlayer(CraftServer.java:537) ~[minecraft_server.jar:git-Paper-329]
at org.bukkit.Bukkit.getPlayer(Bukkit.java:484) ~[minecraft_server.jar:git-Paper-329]
at com.yoonicode.minecraftmanhunt.PluginCommands.UpdateCompass(PluginCommands.java:52) ~[?:?]
at com.yoonicode.minecraftmanhunt.PluginCommands$3.run(PluginCommands.java:299) ~[?:?]
at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:99) ~[minecraft_server.jar:git-Paper-329]
at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1294) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:377) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1209) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:997) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:178) ~[minecraft_server.jar:git-Paper-329]
at java.lang.Thread.run(Thread.java:834) [?:?]
[17:42:53] [Server thread/WARN]: [MinecraftManhunt] Task #10189 for MinecraftManhunt v1.3.3.2 generated an exception
java.lang.IllegalArgumentException: Name cannot be null
at org.apache.commons.lang.Validate.notNull(Validate.java:192) ~[minecraft_server.jar:git-Paper-329]
at org.bukkit.craftbukkit.v1_16_R3.CraftServer.getPlayer(CraftServer.java:537) ~[minecraft_server.jar:git-Paper-329]
at org.bukkit.Bukkit.getPlayer(Bukkit.java:484) ~[minecraft_server.jar:git-Paper-329]
at com.yoonicode.minecraftmanhunt.PluginCommands.UpdateCompass(PluginCommands.java:52) ~[?:?]
at com.yoonicode.minecraftmanhunt.PluginCommands$3.run(PluginCommands.java:299) ~[?:?]
at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:99) ~[minecraft_server.jar:git-Paper-329]
at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1294) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:377) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1209) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:997) ~[minecraft_server.jar:git-Paper-329]
at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:178) ~[minecraft_server.jar:git-Paper-329]
at java.lang.Thread.run(Thread.java:834) [?:?] 
```
These happen on `/start`. Personally I don't think they're relevant to the issue, but I wouldn't really know.

---

Thank you for making this plugin. It's the best one so far!

**Comment by yummypasta** [comment]
Hi @SeerLite,

Thanks for using the plugin. The enchanted glow effect is actually not an intentional feature— behind the scenes, the plugin uses Lodestone compasses to make them work in the nether, so that's why the glow depends on the dimension. I will look into how to make the enchanted effect consistent.

**Comment by yummypasta** [comment]
Fixed in [release v1.3.3.5](https://github.com/yummypasta/MinecraftManhunt/releases/tag/v1.3.3.5).

**Comment by SeerLite** [comment]
Woah that was really quick! Thank you so much, this release is gonna make the experience so much better!
